### PR TITLE
MHR transfer registration report updates

### PR DESCRIPTION
**Pull request by doug-lovett**
*Issue #:* /bcgov/entity#15986

*Description of changes:*
- 15986 verification report and staff report updates.
- 15987 cover letter report updates.
- Examples attached to the tickets.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: mhr_api/report-templates/template-parts/registration/owners.html [hunk 1]
```diff
@@ -56,8 +56,10 @@
                             {% if party.individualName.middle is defined %}&nbsp;{{ party.individualName.middle }}{% endif %}
                         {% endif %}
                     </div>
-                    {% if party.suffix is defined and party.suffix != '' %}
-                    <div class="pt-2">{{ party.suffix|upper }}</div>
+                    {% if party.description is defined and party.description != '' %}
```
Comment (dimak1) [inline mhr_api/report-templates/template-parts/registration/owners.html:59]: `description` field will have the same data in `suffix` for transfers (i saw in one of the PRs) so you got that covered with if/else condition. Just not sure what's the difference.

**Review by dimak1 — APPROVED** [review]

**Review by cameron-eyds — APPROVED** [review]
Looks great!

File: mhr_api/report-templates/template-parts/registration/owners.html (hunk 1 above)
Comment (doug-lovett) [reply mhr_api/report-templates/template-parts/registration/owners.html:59]: Yes, the suffix is retained for report generation of legacy registrations; description is for new registrations and post data migration.

Comment (dimak1) [reply mhr_api/report-templates/template-parts/registration/owners.html:59]: Great, thanks for the clarification.

**Review by owillborn — APPROVED** [review]
Tests look solid, LGTM

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1299-6owstu3p.web.app

**Pull request merged**